resource/ckb-miner.toml

```diff
@@ -58,4 +61,5 @@ worker_type = "EaglesongSimple" # {{
 threads     = 1 # {{
 # dev => delay_type = "Constant"\nvalue = 5000
 # testnet => threads     = 1\nextra_hash_function = "Blake2b"
+# staging => threads     = 1\nextra_hash_function = "Blake2b"
```

Comment:
I think we should use same pow engine on staging as mainnet, it won't be DoSed by asic miner since it's a private env.